wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -25,8 +48,24 @@ class WP_REST_Customize_Changesets_Controller extends WP_REST_Controller {
 	public function __construct() {
 		$this->namespace = 'customize/v1';
 		$this->rest_base = 'changesets';
+	}
 
-		// @todo meta?
+	/**
+	 * Ensure customize manager.
+	 *
+	 * @param string $changeset_uuid UUID.
+	 * @return WP_Customize_Manager Manager.
+	 * @global WP_Customize_Manager $wp_customize
+	 */
+	public function ensure_customize_manager( $changeset_uuid = null ) {
+		global $wp_customize;
+		if ( empty( $wp_customize ) || $wp_customize->changeset_uuid() !== $changeset_uuid ) {
+			$wp_customize = new WP_Customize_Manager( compact( 'changeset_uuid' ) ); // WPCS: global override ok.
```

Comment:
@westonruter Do you think that at some point in the future it would make sense for the `WP_Customize_Manager` to be able to switch between changesets?